### PR TITLE
Feature: export types

### PR DESCRIPTION
**Pull request by luhc228**
ref: https://github.com/ice-lab/icejs/pull/272

### CODE REVIEW

File: packages/plugin-store/src/template/types.ts.ejs [hunk 1]
```diff
@@ -1,4 +1,14 @@
-import * as types from '../appStore';
+import { IcestoreRootState, IcestoreDispatch } from '@ice/store';
+import { appModel } from '.';
```
Comment (chenbin92) [inline packages/plugin-store/src/template/types.ts.ejs:2]: `from '.'` ?

File: packages/plugin-router/src/types/router.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { History } from 'history';
+import { IgnoreOptions } from './collector';
+import { IRouterConfig } from '.';
```
Comment (chenbin92) [inline packages/plugin-router/src/types/router.ts:3]: 改成 `from './index'` 吧，可读性好一点

Comment (luhc228) [reply packages/plugin-router/src/types/router.ts:3]: fixed

File: packages/plugin-store/src/template/types.ts.ejs (hunk 1 above)
Comment (luhc228) [reply packages/plugin-store/src/template/types.ts.ejs:2]: fixed。已修改为 './index'

**Review by ClarkXia — APPROVED** [review]

**Review by chenbin92 — APPROVED** [review]

**Pull request merged**